src/geom/Transform.cc

```diff
@@ -252,16 +253,25 @@ void Transform<FromEndpoint, ToEndpoint>::write(OutputArchiveHandle &handle) con
     Transform<FromEndpoint, ToEndpoint>::then<NextToEndpoint>(          \
             Transform<ToEndpoint, NextToEndpoint> const &next, bool) const;
 
-#define INSTANTIATE_TRANSFORM(FromEndpoint, ToEndpoint)                                \
-    template class Transform<FromEndpoint, ToEndpoint>;                                \
-    template std::ostream &operator<<<FromEndpoint, ToEndpoint>(                       \
-            std::ostream &os, Transform<FromEndpoint, ToEndpoint> const &transform);   \
-    namespace {                                                                        \
-    TransformFactory<FromEndpoint, ToEndpoint> registration##FromEndpoint##ToEndpoint( \
-            Transform<FromEndpoint, ToEndpoint>::getShortClassName());                 \
-    }                                                                                  \
-    INSTANTIATE_OVERLOADS(FromEndpoint, ToEndpoint, GenericEndpoint)                   \
-    INSTANTIATE_OVERLOADS(FromEndpoint, ToEndpoint, Point2Endpoint)                    \
+#define INSTANTIATE_TRANSFORM(FromEndpoint, ToEndpoint)                                    \
+    } /* namespace geom */                                                                 \
```

Comment:
Same question as above.

Reply:
This is the one instantiation that I could not figure out how to really clean up. I need to provide the `FromEndpoint` and `ToEndpoint` arguments without a namespace in order for `registration##FromEndpoint##ToEndpoint` to work correctly, but I also need to get out of the `geom` namespace in order to instantiate `PersistableFacade`. 

I did get rid of the "table" and "io" namespaces here and everywhere else that I instantiate a `PersistableFacade` and I think that makes the code a bit cleaner.